test/TestProjects/MgmtMockAndSample/tests/Generated/Samples/Sample_DeletedVaultCollection_RetrieveADeletedVault1.cs

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using System;
+using System.Threading.Tasks;
+using Azure.Core;
+using Azure.Identity;
+using Azure.ResourceManager;
+using Azure.ResourceManager.Resources;
+
+namespace MgmtMockAndSample
+{
+    public partial class Sample_DeletedVaultCollection_RetrieveADeletedVault
```

Comment:
As we show the sample of 'Exists', shall we still call it RetrieveADeletedVault

Reply:
This name is the name of the example which is defined in the swagger. And this is not part of the final example